back/engines/commercial/admin_api/spec/acceptance/tenants_spec.rb

```diff
@@ -116,16 +116,6 @@
     end
   end
 
-  delete 'admin_api/tenants/:tenant_id' do
-    example_request 'Deleting a tenant', document: false do
-      original_host = tenant.host
-
-      assert_status 200
-      expect(tenant.reload.deleted_at).not_to be_nil
```

Comment:
Good point. I assumed that since I added this test when I added the suffix to deleted tenants that the whole test would now be unnecessary, but actually we should have a test of tenant deletion (just without testing the host has changed) :+1: